spec/controllers/api/public/v1/courses_controller_spec.rb

```diff
@@ -70,6 +74,46 @@
           expect(provider_id).to eq(provider.id)
         end
       end
+
+      context "with sorting" do
```

Comment:
Just noticed that `with includes` above tests how the JSON API renderer works, it's response, etc. This section only tests it calls `CourseSearchService` the right way. Minor point, could be worth aligning.

Reply:
Yeah this was intentional. The `CourseSearchService` is already tested quite extensively. I didn't see the value in duplicating those here in the controller. I am relying on that search class to do its job but asserting it gets the right params to perform that task.

I suppose we could lead with that same assumption for the include scenario but I want to sanity check the include params are being whitelisted properly by the JSON API library we use and that it returns the correct data back.